build/azure-pipelines/product-build.yml

```diff
@@ -6,6 +6,9 @@ resources:
   - container: snapcraft
     image: snapcore/snapcraft:stable
 
+variables:
```

Comment:
@deepak1556 this should not be part of the squashed commit as it was only done for exploration build